backend/src/main/java/com/example/backend/entity/Route.java

```diff
@@ -1,17 +1,20 @@
 package com.example.backend.entity;
 
+import java.util.ArrayList;
 import java.util.HashMap;
 
 public class Route {
 
     HashMap<Integer, Vehicle> vehicles;
     HashMap<String, Stop> stops;
     String routeTag;
+    ArrayList<RouteDirection> routeDirections;
```

Comment:
On second thought, what would you think about storing directions as a map of strings to RouteDirections? It would make accessing direction objects easier

Reply:
(There's not much need for route directions to be ordered anyway)

Reply:
Yeah that makes more sense. I've gone ahead and changed it so that it stores a mapping of the direction tag to the RouteDirection object.